Source/WebCore/html/HTMLImageLoader.cpp

```diff
@@ -72,13 +72,13 @@ void HTMLImageLoader::dispatchLoadEvent()
     element().dispatchEvent(Event::create(errorOccurred ? eventNames().errorEvent : eventNames().loadEvent, Event::CanBubble::No, Event::IsCancelable::No));
 }
 
-void HTMLImageLoader::notifyFinished(CachedResource&, const NetworkLoadMetrics& metrics)
+void HTMLImageLoader::notifyFinished(CachedResource&, const NetworkLoadMetrics& metrics, bool loadWillContinueInAnotherProcess)
```

Comment:
This seems layering violation that a WebCore function knows/cares about process. Maybe something like `shouldNotifyClient`.

Reply:
LoadWillContinueInAnotherProcess

Reply:
WebCore is learning more and more about the existence of other processes, though it doesn't communicate with them directly.